modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -145,10 +145,10 @@ $ aws kms get-key-policy --key-id <key_id_or_arn> --policy-name default --output
             "Effect": "Allow",
             "Principal": {
                 "AWS": [
-                    "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Support-Role", <1>
-                    "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Installer-Role",
-                    "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Worker-Role",
-                    "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-ControlPlane-Role",
+                    "arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role", <1>
+                    "arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role",
+                    "arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role",
+                    "arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role",
                     "arn:aws:iam::<aws_account_id>:role/<cluster_name>-xxxx-openshift-cluster-csi-drivers-ebs-cloud-credent" <2>
```

Comment:
The operator role arn name is not fixed. It will change according to the length of the prefix ：<cluster_name>-xxxx